src/content/learn/you-might-not-need-an-effect.md

```diff
@@ -1675,7 +1677,7 @@ label, textarea { margin-bottom: 10px; display: block; }
 
 <Solution>
 
-The `showForm` state variable determines whether to show the form or the "Thank you" dialog. However, you aren't sending the message because the "Thank you" dialog was _displayed_. You want to send the message because the user has _submitted the form._ Delete the misleading Effect and move the `sendMessage` call inside the `handleSubmit` event handler:
+`showForm` state değişkeni formun veya "Thank you" iletişim kutusunun gösteriliğ gösterilmeyeceğine karar verir. Ancak "Thank you" iletişim kutusu _görüntülendiği_ için mesajı gönderemiyorsunuz. Kullanıcının  _formu göndermesi_ nedeniyle mesajı göndermek istiyorsunuz. Yanıltıcı Efekti silin ve `sendMessage` çağrısını `handleSubmit` olay yöneticisi içerisine taşıyın:
```

Comment:
Ilk cumle: 

`showForm` state değişkeni, formun mu yoksa "Thank you" mesajının mı gösterileceğini belirler.  

bir de:

 `gönderemiyorsunuz` -> `göndermiyorsunuz`